src/libponyrt/sched/scheduler.h

```diff
@@ -56,6 +56,8 @@ struct scheduler_t
   uint32_t ack_count;
   mutemap_t mute_mapping;
 
+  uint64_t quiescence_cycles;
+
```

Comment:
did you verify that the alignment for message queue is still correct?

Reply:
see: "These are accessed by other scheduler threads. The mpmcq_t is aligned." comment

Reply:
@SeanTAllen I think that the compiler was already enforcing 64-bit alignment of mpmcq_t because its first field, mpmcq_t::head, is declared with alignas(64). That being said, I have added a unit test to assert that this is actually the case. I've also added a unit test to double check the type sizes when transporting uint64_t in pony_msgi_t. Please see: 8a3fca2b00a0458bd47b97348dade2a3e0fe5299.